Workbooks/WorkbooksMetadata.json

```diff
@@ -944,5 +944,18 @@
     "templateRelativePath": "NormalizedNetworkEvents.json",
     "subtitle": "",
     "provider": "Microsoft"
+  },
+     {
+  "workbookKey": "WorkspaceAuditingWorkbook",
+  "logoFileName": "paloalto_logo.svg",
+  "description": "Workspace auditing report\r\nUse this report to understand query runs across your workspace.",
+  "dataTypesDependencies": [ "LAQueryLogs" ],
+  "dataConnectorsDependencies": [ "" ],
+  "previewImagesFileNames": [ "WorkspaceAuditgingWhite.png", "WorkspaceAuditingBlack.png" ],
```

Comment:
typo in auditing, you wrote audit**g**ing

Reply:
fixed